content/tos/index.md

```diff
@@ -0,0 +1,113 @@
+2. Eligibility.
+   1. To access or use the Service, you must have a valid email address and a valid form of payment. You acknowledge and agree that you will only create one account per email address. During the user registration process for the Service, you will be required to agree to this Agreement.
+   2. During the registration process, you will be provided an Urbit ship if you do not already have one. You will be responsible for custody of the ownership keys of your ship. If you lose these, we cannot recover them for you, and you could lose access to your ship and your data permanently.
+   3. The Service is intended solely for individuals who are at least 18 years of age or older. Any registration on or use of the Service by any individual who does not meet the foregoing requirement is not authorized to register for or use the Service. By using the Service, you represent and warrant that you meet the foregoing requirement.
+3. Registration.
+   1. When you register for a user account, you will be asked to disclose certain personal information, including your name, your business address, the company or entity that you represent (if applicable), your billing information, your email address, and your selected password (“Account Information”), all of which will be subject to our Privacy Policy (available here, and provide us management credentials, but not ownership credentials, to your Urbit ship.
+   2. You agree to provide us with true, accurate and complete Account Information requested in the registration process. You also agree to update such Account Information promptly as necessary to keep it current and accurate.
+4. Security and Data Privacy.
+   1. Without limiting Section 13 or your obligations under Section 5, we will implement reasonable and appropriate measures designed to help you secure the software, data, text, audio, video or images that you upload to the Service (“Your Content”) against accidental or unlawful loss, access or disclosure.
+   2. We will not access or use Your Content except as necessary to maintain or provide the Service, or as necessary to comply with the law or a binding order of a governmental body. We will not (a) disclose Your Content or Your Ethereum address to any government or third party, except as necessary to comply with the law or a binding order of a governmental body. Unless it would violate the law or a binding order of a governmental body, we will give you notice of any legal requirement or order referred to in this Section 4.2. We will only use your Account Information in accordance with our Privacy Policy, and you consent to such usage. The Privacy Policy does not apply to Your Content or Your Ethereum address.
```

Comment:
> We will not (a) disclose Your Content or Your Ethereum address to any government or third party

This does not cover the Urbit ID, from which the corresponding Ethereum address(es) can be trivially obtained. Unsure about the intended scope of this clause, but maybe it should be included?

(Also, there is no (b) in this section.)

Reply:
> Unless it would violate the law or a binding order of a governmental body, we will give you notice of any legal requirement or order referred to in this Section 4.2.

Is there any planned Tlon warrant canary? 👼

Reply:
We've been thinking quite a lot about a warrant canary, but haven't settled on the right scope yet. The bottom line is that the law isn't settled, and we don't have the resources or interest in holding ourselves out as a test case.

Reply:
I agree we should include Urbit ID's in the disclosure policy.